### ISSUE TITLE
Cursive crashes trying to open a project with a :keyword as the project.clj version

### ISSUE THREAD

**Issue by timgilbert** [cursive-ide/cursive#1999]
I'm trying to use the `lein-v` project to derive my version numbers from git tags instead of a literal string in the project.clj:

https://github.com/roomkey/lein-v

It works fine from the command-line, but when I try to import the directory as an IntelliJ project I get an exception about casting a keyword to a string.

This seems to only be a problem with import. I had an already-imported file with a regular string version number, and when I changed it to `:lein-v` after the fact, Cursive seemed to have no issues working with the project, both as a top-level project and as a checkout dependency of another project.

As a workaround, I just turned `:lein-v` into `":lein-v"`, imported the project, and then changed it back to `:lein-v`, which worked fine. It seemed like it was probably worth opening a ticket though.

**Comment by jdf-id-au** [comment]
I worked around this by dumping lein-v and using:
```
(def version
  (-> (clojure.java.shell/sh "git" "describe" "--always")
      :out
      clojure.string/trim))

(defproject projname #=(eval version)...
```
Less fancy, but one less dependency!

**Comment by cursive-ide** [comment]
I've just fixed this as well as roomkey/lein-v#26. It's a fairly sensitive change, so it'll be in the next release cycle (1.10.2).
